Automattic/wp-calypso
# Replace Yoast banner with Blaze banner on /marketing/traffic page

#### Proposed Changes

* Replace the Yoast SEO banner on marketing/traffic with a banner for Blaze.

Before | After
--|--
![marketing-traffic-before](https://user-images.githubusercontent.com/140841/208168257-07767eee-d58c-431e-a8ae-720efab72083.png)  |  ![marketing-traffic-after](https://user-images.githubusercontent.com/140841/208168278-faafc091-cd6a-4e51-a5b4-84b7f7452c8c.png)



#### Testing Instructions



* Load this PR.
* Ensure clicking the banner CTA takes you to `https://wordpress.com/advertising/{blog-domain}`
* Ensure Tracks are being recorded for impressions and clicks
* Make sure banner appears on Simple, Atomic, and Jetpack connected sites
* Ensure the banner matches the Figma. pb5gDS-2fR-p2

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/70835
